tag-historian/nitaghistorian.yml

```diff
@@ -322,7 +322,7 @@ definitions:
         minimum: 0
         maximum: 1
         example: 1
-      errors:
+      error:
```

Comment:
Shouldn't the server version stay the same since we haven't changed its implementation?

Reply:
Yep! I mistakenly thought that the swagger doc for server was inaccurate too.